src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateNuSpec.cs

```diff
@@ -193,6 +193,7 @@ private Manifest CreateManifest()
         private List<ManifestFile> GetManifestFiles()
         {
             return (from f in Files.NullAsEmpty()
+                    where !f.GetMetadata(Metadata.FileTarget).StartsWith("none", StringComparison.OrdinalIgnoreCase)
```

Comment:
Angle brackets don't work since our path APIs reject them.  I used $'s since that'll be less likely to collide with any use case.